### DISCUSSION TITLE
Problems with installation

### DISCUSSION THREAD

**Discussion by dwlr09** [Security-Onion-Solutions/securityonion#4900] [Unsupported Versions] [answered]
We are currently building our environment and on every box, we get an error message. When we check /root/error.log we get this result:
1 [ERROR   ] 'pkg.clean_metadata': False

Does anyone have a fix action for this?  We have tried on 2.3.6 and 2.3.5 same result.

**Comment by SOnewbie** [comment]
> We are currently building our environment and on every box, we get an error message. When we check /root/error.log we get this result: 1 [ERROR ] 'pkg.clean_metadata': False
> 
> Does anyone have a fix action for this? We have tried on 2.3.6 and 2.3.5 same result.

I get this on 2.3.61, 2.3.70 and 2.3.80 no idea how to fix this either.

**Comment by wompRS** [comment]
I've gotten this error on each of the last 3 installation ISOs as well. Not sure what the cause is. I am running in VMware using the Centos 7 configuration in the drop-down menu.

**Reply by AnyrunMalware** [reply]
I am getting the same, did you find out what the issue was?

**Comment by dougburks** [comment, marked as answer]
What options are you choosing during Setup?

Is this an airgap installation or do you have Internet access?

**Reply by dwlr09** [reply]
Mine was airgap. Once we got the 2.3.62 iso it didn't happen again. Also we had to sync the clocks on our heavy's and masters. Since we didn't use an NTP the internal clock on the servers was more than 5 minutes apart. After that we haven't had any issues since

**Reply by dougburks** [reply]
@dwlr09 OK, since you are the original poster of this discussion, I am going to mark this as answered.

For any others that may be experiencing this issue, please create a separate discussion to avoid confusion (as described in https://github.com/Security-Onion-Solutions/securityonion/discussions/1720). Thanks!

**Comment by wompRS** [comment]
It has internet access. Manager + Search node. It does the same thing when
trying standalone mode.

**Comment by dwlr09** [comment]
Not really an answer we just got another ISO. Also since we aren’t using an
 NTP server we used some support hours to figure out the salt commands to
get the heavy nodes within 5 minutes and the errors cleared up

**Comment by dougburks** [comment]
This discussion has been marked as answered (see above). For any others that may be experiencing this issue, please create a separate discussion to avoid confusion (as described in #1720). In that separate discussion, please include the output of `sudo salt-call pkg.clean_metadata -l debug`.
